### PR TITLE
Add web UI for Gamestonk Terminal

### PR DESCRIPTION
**Pull request by CoconutMacaroon**
# Description

- [X] Summary of the change / bug fix.
  This adds a version of the terminal that can be viewed in a browser over the network.
- [ ] Link # issue, if applicable.
  Discussed via DMs in Discord with @DidierRLopes 
- [X] Screenshot of the feature or the bug before/after fix, if applicable.
  Screenshot of web UI running in Firefox Developer Edition
- [X] Relevant motivation and context.
  Setting this up on every computer I wanted to access it from was starting to be annoying, so I made a web UI so any device with a browser can use it
- [X] List any dependencies that are required for this change.
  Docker? I think that's already required, though.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
   I did not write any test due to what I added. If you have an idea of how (or even what) would get tested, then feel free to edit this.
* Provide instructions so we can reproduce. 
   Clone my fork. Go into the `gamestonk_terminal_web` folder, run `chmod +x launch`, then do `sudo ./launch`, wait for it to build, and head to `http://localhost:8080` once it is built. Or, `http://host-ip:8080` if you're testing it on a different computer, say, over SSH.
* Please also list any relevant details for your test configuration.
   I'm using Docker on Debian 11 Bullseye with a custom 5.15 Linux kernel, but it should work on stock Debian, too.

# Checklist:

- [ ] Update [our Hugo documentation](https://gamestonkterminal.github.io/GamestonkTerminal/) following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/website).
- [ ] Update our tests following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/scripts).

Not sure exactly what to put in the docs, so I didn't do that. I didn't add tests (see above), and it isn't a script, so it didn't go into a script file.

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [x] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.
       Tests fail with cryptic error messages locally, I'm hoping they work on GitHub though.

### CODE REVIEW

**Comment by CoconutMacaroon** [comment]
@DidierRLopes Can you review this please?

**Comment by DidierRLopes** [comment]
> @DidierRLopes Can you review this please?

Let's make sure tests pass!

File: README.md [hunk 1]
```diff
@@ -158,6 +159,20 @@ This installation type supports both Windows and Unix systems (Linux + MacOS). H
 
 If you are using macOS or other Linux operating systems you can jump the next section <a href="#installing-the-terminal">Installing the terminal</a>.
 
+### Web UI - Docker
```
Comment (DidierRLopes) [inline README.md:162]: Can you add this on the table of contents above?

**Review by DidierRLopes — APPROVED** [review]
Looks good to me!

Let's get plots on this baby!

File: README.md (hunk 1 above)
Comment (CoconutMacaroon) [reply README.md:162]: Sorry, I forgot about the Table of Contents. One moment

**Comment by CoconutMacaroon** [comment]
@DidierRLopes I think I updated the table of contents, can you approve the workflows just to double-check that I didn't make any errors? Thanks! I hope to add charts next week.

**Comment by DidierRLopes** [comment]
#1253 will fix this

**Pull request merged**